### PR TITLE
Fix building of the docs

### PR DESCRIPTION
**Pull request by g1itch**
Hi!

These changes should fix the recent failed builds on RTFD and [empty pages](https://pybitmessage.readthedocs.io/en/v0.6/autodoc/pybitmessage.bitmessageqt.settings.html) under `pybitmessage.bitmessageqt` section.

[Preview](https://pybitmessage-test.readthedocs.io/en/doc-fix/autodoc/pybitmessage.bitmessageqt.settings.html)

### CODE REVIEW

**Review by PeterSurda — APPROVED** [review]
I don't understand it but it doesn't seem to affect anything that I worry breaking so I'm ok with it.

**Comment by g1itch** [comment]
> I don't understand it but it doesn't seem to affect anything that I worry breaking so I'm ok with it.

Don't you see the build failures currently on pybitmessage.readthedocs.io?

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 m2r
 sphinxcontrib-apidoc
+docutils<=0.17.1
```
Comment (g1itch) [inline docs/requirements.txt:3]: Hmm, it seems that actually this hunk fixes the RTD build, not only my local test with tox.

**Comment by g1itch** [comment]
`tox -r -e py27-doc` should reproduce the issue

**Comment by PeterSurda** [comment]
> Don't you see the [build failures](https://readthedocs.org/projects/pybitmessage/builds/) currently on pybitmessage.readthedocs.io?

What I meant was I'm not sufficiently familiar with tox or sphinx to provide meaningful feedback.

**Pull request merged**